xlint

```diff
@@ -389,6 +389,7 @@ for argument; do
 	if [ "$template" ]; then
 	exists_once "$template"
 	scan 'short_desc=.*\."' "unwanted trailing dot in short_desc"
+	scan 'short_desc=.*\ "' "unwanted trailing space in short_desc"
```

Comment:
```suggestion
	scan 'short_desc=.*[[:space:]]"' "unwanted trailing space in short_desc"
```

this will also match tab, newline, vertical tab, form feed, and carriage return